ibm_db2/assets/configuration/spec.yaml

```
@@ -56,6 +56,13 @@ files:
           has to be accessible and readable by the agent user.
         value:
           type: string
+      - name: connection_timeout
+        description: |
+          Connection timeout in seconds
+        value:
+          type: integer
+          example: 1
```

I would bump this a little higher

```suggestion
          example: 10
```